iadvize-client/src/main/kotlin/ai/tock/iadvize/client/constants.kt

```diff
@@ -30,6 +30,8 @@ const val IADVIZE_PASSWORD_AUTHENTICATION = "iadvize_password_authentication"
 const val IADVIZE_PROXY_HOST = "iadvize_proxy_host"
 const val IADVIZE_PROXY_PORT = "iadvize_proxy_port"
 
+const val ADVIZE_CREDENTIALS_PROVIDER_TYPE = "iadvize_credentials_provider_type"
```

Comment:
```suggestion
const val IADVIZE_CREDENTIALS_PROVIDER_TYPE = "iadvize_credentials_provider_type"
```

faute de frappe probablement